spec/rails_helper.rb

```diff
@@ -7,6 +7,7 @@
 require "rspec/rails"
 require "capybara/rspec"
 require "capybara-screenshot/rspec"
+require "database_cleaner"
```

Comment:
This is still a problem for me against latest master.  I don't know what else is requiring database cleaner without this line.  If it works for you, I'd be interested in seeing what requires it if you put a raise at the top of the database cleaner gem entry point file and post the stacktrace.